docs/organizer-guide/confs/website.rst

```diff
@@ -0,0 +1,293 @@
+    * ``main``: the human readable conference dates and location,
+      e.g. ``"**May 3-5, 2020, in Portland, Oregon**"``.
+    * ``short``: the short human readable dates, e.g. ``May 3-5, 2020``.
+    * ``tickets_live``: a rough human readable date when tickets go on sale,
+      e.g. ``January 2020``.
+    * ``month``: the month in which the conference is held, e.g. ``May``.
+    * ``total_talk_days``: the number of days that have talks, e.g. ``2``.
+      Used to automatically read the schedule.
+    * ``day_one``, ``day_two``, etc. These are actually events, not days. Each "day" has:
+        * ``event``: the name of the event, like ``Hike``, ``Writing Day`` or
+          ``Main Conference``.
+        * ``date``: the short human readable date, e.g. ``May 2`` or
+          ``May 4-5``.
+        * ``summary``: a human readable summary of the event that day.
+        * ``icon``: the icon used for this event, e.g. ``hike`` or
+          ``conference``.
+        * ``dotw``: the day(s) of the week for this event, e.g. ``Saturday``
+          or ``Monday/Tuesday``.
+* ``about``: general conference background. Contains:
+    * ``attendees``: the number of attendees.
+    * ``summary``: a summary text for the conference.
+    * ``venue``: a human readable textual description of the venue location.
+    * ``photos``: a link to the conference photos (typically on Flickr).
+    * ``mainroom``: the name of the main room.
+    * ``unconfroom``: the name of the unconference room.
+    * ``projector_ratio``: the ratio for the projector, e.g. ``16:9``.
+* ``cfp``: call for papers details. Contains:
+    * ``url``: the URL to a google form with the CFP.
+    * ``ends``: a human readable date of when the CFP ends,
+      e.g. ``31 January, 2020``.
+    * ``notification``: a human readable date of when accepted speakers
+      will be notified.
+* ``job_fair.location``: a description of the location of the job fair.
+
+Then, there are a number of true/false flags. Some of these don't change
+as the conference is being planned. These are:
+
+* ``flaghashike``: does the conference have a hike?
+* ``flaghasboat``: does the conference have a boat ride?
+* ``flagwelcomewagon``: does the conference have a welcome wagon?
+
+Others will change over time:
+
+* ``flaglanding``: is the conference website only a landing page?
+  Typically used for an early announcement of conference dates.
+* ``flagticketsonsale``: are tickets on sale at this time?
+* ``flagsoldout``: is the conference sold out?
+* ``flagcfp``: is the CFP currently open?
+* ``flagspeakersannounced``: have speakers been announced?
+  This flag requires the ``_data/<shortcode>-<year>-sessions.yaml``
+  file to exist, which lists the speakers/sessions.
+* ``flaghasschedule``: is the schedule ready?
+  This flag requires the``_data/<shortcode>-<year>-schedule.yaml``
+  file to exist, which contains the schedule.
+* ``flaglivestreaming``: is a live stream currently running?
```

Comment:
probably should change this to "is a live stream available", since the page needs to go live before we start the conf..